graphql/database/search/getSearchEngine.js

```diff
@@ -2,9 +2,12 @@ import { find } from 'lodash/collection'
 import searchEngines from './searchEngines'
 import { DatabaseItemDoesNotExistException } from '../../utils/exceptions'
 
-const getSearchEngine = async id => {
+const getSearchEngine = async (id, ignoreUnfound = false) => {
```

Comment:
A better pattern is for the calling code to try/catch and specifically handle the DatabaseItemDoesNotExistException.

Reply:
Example: https://github.com/gladly-team/tab/blob/587b2fc1f960924bb0e5871451dfb2164d4259cb/graphql/database/users/rewardReferringUser.js#L45